app/_includes/nav-v2.html

```diff
@@ -90,7 +90,10 @@
         </li>
       </ul>
       <a id="top-cta" href="https://incubator.konghq.com/?utm_source=docs.konghq.com" class="navbar-button" target="_blank">
-        Early Access
+        Personalized Demo
```

Comment:
```suggestion
        Get A Demo
```

Reply:
@fabianrbz had a parallel request to change text on this button, so adding in here as proposed change.